utils/docker_nlt.sh

```diff
@@ -6,8 +6,29 @@
 
 set -e
 
+set -x
+
 . utils/sl/setup_local.sh
 
-./utils/setup_daos_admin.sh
+sudo --preserve-env=SL_PREFIX ./utils/setup_daos_admin.sh
+
+TMP_DIR=$(mktemp -d)
+
+cp utils/node_local_test.py utils/nlt_server.yaml utils/nlt_agent.yaml .build_vars.json "$TMP_DIR"
+cp src/tests/ftest/cart/util/cart_logparse.py src/tests/ftest/cart/util/cart_logtest.py "$TMP_DIR"
+
+pushd "$TMP_DIR"
+
+set +e
+
+sudo ./node_local_test.py --no-root --memcheck no --server-debug WARN "$@"
```

Comment:
Is this going to leave files behind in the workspace, owned by root, that Jenkins cannot clean up as it does not run as root?

Reply:
No, whilst this PR predates that ticket I've addressed the issue here.  What the code is doing now is running the sudo with the current directory being a location in the docker image, then copying out the results files without sudo afterwards.

The end result should be that no files are created owned by root in the workspace, although the downside is that if there's a timeout then there will probably be no results files available to Jenkins.